<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="users_list">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>11</y>
      <width>271</width>
      <height>541</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="send_btn">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>500</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="msg_input">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>500</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="messages_list">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>301</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
